src/common/components/Layouts/LayoutWithSearch/__snapshots__/index.spec.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Component/LayoutWithSearch test Should match snapshots 1`] = `ShallowWrapper {}`;
```

Comment:
it seem not capture a real rendering a component, please check

Reply:
fixed at f4a6fb4f1c5a3f92abc8256fafb9511c28d0eab8